File: pkg/codegen/testing/utils/testdata/aws-static-website-pp/python/aws-static-website.py
import pulumi
import pulumi_aws_static_website as aws_static_website

website_resource = aws_static_website.Website("websiteResource",
    site_path="string",
    index_html="string",
    cache_ttl=0,
    cdn_args={
        "cloudfront_function_associations": [{
            "event_type": "string",
            "function_arn": "string",
        }],
        "forwarded_values": {
            "cookies": {
                "forward": "string",
                "whitelisted_names": ["string"],
            },
            "query_string": False,
            "headers": ["string"],
            "query_string_cache_keys": ["string"],
        },
        "lambda_function_associations": [{
            "event_type": "string",
            "lambda_arn": "string",
            "include_body": False,
        }],
    },
    certificate_arn="string",
    error404="string",
    add_website_version_header=False,
    price_class="string",
    atomic_deployments=False,
    subdomain="string",
    target_domain="string",
    with_cdn=False,
    with_logs=False)

File: pkg/codegen/testing/utils/testdata/aws-static-website-pp/dotnet/aws-static-website.cs
using System.Collections.Generic;
using System.Linq;
using Pulumi;
using AwsStaticWebsite = Pulumi.AwsStaticWebsite;

return await Deployment.RunAsync(() => 
{
    var websiteResource = new AwsStaticWebsite.Website("websiteResource", new()
    {
        SitePath = "string",
        IndexHTML = "string",
        CacheTTL = 0,
        CdnArgs = new AwsStaticWebsite.Inputs.CDNArgsArgs
        {
            CloudfrontFunctionAssociations = new[]
            {
                new Aws.CloudFront.Inputs.DistributionOrderedCacheBehaviorFunctionAssociationArgs
                {
                    EventType = "string",
                    FunctionArn = "string",
                },
            },
            ForwardedValues = new Aws.CloudFront.Inputs.DistributionDefaultCacheBehaviorForwardedValuesArgs
            {
                Cookies = new Aws.CloudFront.Inputs.DistributionDefaultCacheBehaviorForwardedValuesCookiesArgs
                {
                    Forward = "string",
                    WhitelistedNames = new[]
                    {
                        "string",
                    },
                },
                QueryString = false,
                Headers = new[]
                {
                    "string",
                },
                QueryStringCacheKeys = new[]
                {
                    "string",
                },
            },
            LambdaFunctionAssociations = new[]
            {
                new Aws.CloudFront.Inputs.DistributionOrderedCacheBehaviorLambdaFunctionAssociationArgs
                {
                    EventType = "string",
                    LambdaArn = "string",
                    IncludeBody = false,
                },
            },
        },
        CertificateARN = "string",
        Error404 = "string",
        AddWebsiteVersionHeader = false,
        PriceClass = "string",
        AtomicDeployments = false,
        Subdomain = "string",
        TargetDomain = "string",
        WithCDN = false,
        WithLogs = false,
    });

});

